state/event_table_test.go

```diff
@@ -1165,52 +1166,59 @@ func TestEventTable_SelectLatestEventsBetween_MissingPrevious(t *testing.T) {
 	}
 
 	testcases := []struct {
+		Desc            string
 		FromIDExclusive string
 		ToIDInclusive   string
-		Limit           int
 		ExpectIDs       []string
 	}{
 		{
+			Desc:            "has no gaps - should omit nothing",
 			FromIDExclusive: "start",
 			ToIDInclusive:   "C",
 			ExpectIDs:       []string{"C", "B", "A"},
 		},
 		{
+			Desc:            "ends with missing_previous - should include the end only",
 			FromIDExclusive: "start",
-			ToIDInclusive:   "D",
-			ExpectIDs:       []string{"D"},
+			ToIDInclusive:   "D-gap",
+			ExpectIDs:       []string{"D-gap"},
 		},
 		{
+			Desc:            "single event, ends with missing_previous - should include the end only",
 			FromIDExclusive: "C",
-			ToIDInclusive:   "D",
-			ExpectIDs:       []string{"D"},
+			ToIDInclusive:   "D-gap",
+			ExpectIDs:       []string{"D-gap"},
 		},
 		{
-			FromIDExclusive: "D",
+			Desc:            "single event, start is missing_previous - should include the end only",
+			FromIDExclusive: "D-gap",
 			ToIDInclusive:   "E",
 			ExpectIDs:       []string{"E"},
 		},
 		{
+			Desc:            "covers two consecutive gaps - should include events from the second gap onwards.",
 			FromIDExclusive: "E",
 			ToIDInclusive:   "J",
-			ExpectIDs:       []string{"J", "I", "H"},
+			ExpectIDs:       []string{"J", "I", "H-gap"},
 		},
 		{
+			Desc:            "covers three gaps, the latter two consecutive - should only return from the second gap onwards",
 			FromIDExclusive: "B",
 			ToIDInclusive:   "I",
-			ExpectIDs:       []string{"I", "H"},
+			ExpectIDs:       []string{"I", "H-gap"},
 		},
 	}
 
 	for _, tc := range testcases {
+		// We're using the notation (X, Y] for a half-open interval excluding X but including Y.
```

Comment:
I personally prefer [rust's notation](https://doc.rust-lang.org/std/ops/struct.RangeInclusive.html) of `X..=Y`

Reply:
`X..=Y` includes X, but I want to exclude X here.